libvmi/driver/xen/xen_events_48.c

```diff
@@ -1335,11 +1387,15 @@ void xen_events_destroy_48(vmi_instance_t vmi)
     (void)xen_shutdown_single_step_48(vmi);
     (void)xen->libxcw.xc_set_mem_access(xch, dom, XENMEM_access_rwx, ~0ull, 0);
     (void)xen->libxcw.xc_set_mem_access(xch, dom, XENMEM_access_rwx, 0, xen->max_gpfn);
+#if defined(I386) || defined(X86_64)
     (void)xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR0, false, false, false);
     (void)xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR3, false, false, false);
     (void)xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR4, false, false, false);
     (void)xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_XCR0, false, false, false);
     (void)xen->libxcw.xc_monitor_software_breakpoint(xch, dom, false);
+#elif defined(ARM32) || defined(ARM64)
+    (void)xen->libxcw.xc_monitor_privileged_call(xch, dom, false);
+#endif
     (void)xen_set_guest_requested_event_48(vmi, 0);
     (void)xen_set_cpuid_event_48(vmi, 0);
```

Comment:
The cpuid and debug events are also x86 specific.

Reply:
You're right I will fix it